src/flyte/_run.py

```diff
@@ -437,8 +448,8 @@ async def _run_local(self, obj: TaskTemplate[P, R], *args: P.args, **kwargs: P.k
                 checkpoint_path=internal_ctx().raw_data.path,
             ),
             code_bundle=None,
-            output_path=self._metadata_path,
-            run_base_dir=self._metadata_path,
+            output_path=str(output_path),
+            run_base_dir=str(metadata_path),
             version="na",
             raw_data_path=internal_ctx().raw_data,
```

Comment:
```suggestion
            raw_data_path= raw_data_path,
```
I think we should use `raw_data_path` here?

Reply:
fixed, thank you and good catch